src/lib.rs

```diff
@@ -16,8 +16,15 @@
 
 #[macro_use]
 extern crate alloc;
+#[cfg(not(feature = "fuzzing"))]
 extern crate lang_items;
 extern crate libtock_core;
 extern crate libtock_drivers;
+#[macro_use]
+extern crate cbor;
+#[macro_use]
```

Comment:
When you run 

> ./deploy.py --opensk --board=nrf52840dk

you get

> error: unused `#[macro_use]` import

(You also see those with `./run_desktop_tests.sh`.)